### PR TITLE
Fixed singular-plural issue in days left/ago

### LINKED ISSUE #1718: Singular/plural form differentiation in "days left/ago"
**Is your feature request related to a problem? Please describe.**
It's very annoying the correct plural form is not used. #1705 was not clearly specced - and thus fixed in a way that removes the differentiation between writing "1 day ago/left" and "2 days ago/left" - and just says "day(s)" for any case.

**Describe the solution you'd like**
Use the correct spelling:  
- day when singular. 
- days when plural. 

Note: for Dutch it's dag/dagen ... so "Nog x dag(en)"/ "y dag(en) geleden"

**Describe alternatives you've considered**
None. This is a small check and improves readability a lot.

**Additional context**

### PR DESCRIPTION
**Pull request by mukul7661**
resolves #1718

### CODE REVIEW

File: src/translations/nl/translation.json [hunk 1]
```diff
@@ -166,8 +166,10 @@
   "noDuration": "Geen duur",
   "dueDate": "Deadline",
   "dueToday": "Vandaag verschuldigd",
-  "dueDatePassed": "Vervaldatum verstreken",
-  "daysLeft": "Nog {{days}} dagen",
+  "dueDatePassed": "{{days}} dagen geleden",
+  "dueDatePassedSingular": "{{days}} dag geleden",
+  "daysLeft": "{{days}} dagen resterend",
```
Comment (tijlleenders) [inline src/translations/nl/translation.json:171]:
```suggestion
  "daysLeft": "Nog {{days}} dagen",
```

File: src/translations/nl/translation.json [hunk 2]
```diff
@@ -166,8 +166,10 @@
...
+  "daysLeftSingular": "{{days}} dag resterend",
```
Comment (tijlleenders) [inline src/translations/nl/translation.json:172]:
```suggestion
  "daysLeftSingular": "Nog {{days}} dag",
```

**Review by tijlleenders — DISMISSED** [review]
Perfect, thanks @mukul7661

**Comment by tijlleenders** [comment]
@Tushar-4781 OK?

**Review by Tushar-4781 — APPROVED** [review]

**Pull request merged**